### ISSUE TITLE
InfoBar: Lack of announcement for Severity level and icon

### ISSUE THREAD

**Issue by gabbybilka** [microsoft/microsoft-ui-xaml#4533]
**Describe the bug**
The Severity level and icon used of an InfoBar is not communicated to users using screen readers.

**Steps to reproduce the bug**
In Narrator, navigate to the InfoBar XCG page and re-open an InfoBar. Only the Title and Message properties are announced without keyboard focus.

**Expected behavior**
The Severity level (Informational/Success/Warning/Error) and icon are communicated with behavior like "Error InfoBar, ErrorBadge icon"

**Open questions**
- Since the Severity level and icon are not tied to a specific UIA field, how can these take Narrator focus?

**Comment by StephenLPeters** [comment]
How is narrator going to get info about the icon? Our icon system doesn't support this today.

**Comment by ranjeshj** [comment]
@gabbybilka  @YuliKl @StephenLPeters   we can make the icon a narrator stop and announce the severity based on the Severity level property. We cannot announce the icon itself but there is always a severity level set on the InfoBar we can use.
